src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -54,14 +54,13 @@ the MIT License. See LICENSE in the project root for license information. -->
                HorizontalAlignment="Left"
                Grid.Row="1"
                SelectionChanged="Pivot_SelectionChanged"
-               Margin="{StaticResource PivotIndentMargin}">
+               Margin="1,0,0,0">

```

Comment:
Why 1 off the left?
Looks like the [Pivot code](https://github.com/microsoft/microsoft-ui-xaml/blob/6d680bc91e67b9fa7a365dc020f24cc2a20b60b5/dev/Pivot/Pivot_themeresources_v2.5.xaml#L16) has a margin of 12 off the left, so I guess 12+1=13 to align with the header?

Reply:
Yep, exactly.